src/components/scene/wakelock.js

```diff
@@ -0,0 +1,15 @@
+var register = require('../../core/component').registerComponent;
+var Wakelock = require('../../../vendor/wakelock/wakelock');
```

Comment:
Why do you want to make it a component then. If it's not removable or not configurable? It's stuff that leaks out of the scene that it's an internal implementation detail.